doc/source/dev/index.rst

```diff
@@ -48,6 +48,7 @@ the organization section.
    contributor/adding_new
    contributor/continuous_integration
    contributor/using_act
+   contributor/triage
```

Comment:
Should be this or we need to move `triage.rst`
```suggestion
   triage
```

Reply:
Oops thanks!